targets/csharp/source/source/PlayFabSettings.cs.ejs

```diff
@@ -30,11 +30,15 @@ namespace PlayFab
         [Obsolete("Moved to PlayFabSettings.staticSettings.VerticalName")]
         public static string VerticalName { get { return staticSettings.VerticalName; } set { staticSettings.VerticalName = value; } }
 
+#if ENABLE_PLAYFABSERVER_API || ENABLE_PLAYFABADMIN_API
         [Obsolete("Moved to PlayFabSettings.staticSettings.DeveloperSecretKey")]
         public static string DeveloperSecretKey { get { return staticSettings.DeveloperSecretKey; } set { staticSettings.DeveloperSecretKey = value; } }
+#endif
 
         [Obsolete("Moved to PlayFabSettings.staticSettings.TitleId")]
         public static string TitleId { get { return staticSettings.TitleId; } set { staticSettings.TitleId = value; } }
+
+#if !DISABLE_PLAYFABCLIENT_API
```

Comment:
I'd put the #ifs inside of the <% if (hasClientOptions) { %> ejs, because otherwise, you'll have:

```
#if !DISABLE_PLAYFABCLIENT_API
#endif
```
In the server output source.
Generate this SDK and make sure you like all the changes.